app/views/sensors/show.html.haml

```diff
@@ -8,6 +8,12 @@
 %p
   %strong Address:
   = @sensor.address
+
+- if user_signed_in? && @sensor.animal_id?
```

Comment:
Yeah, I thought it was better to show it only to authenticated users, because the animal_id is not relevant for a standard user.